python/db/items.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS items (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT NOT NULL,
+    category_id INTEGER NOT NULL,
+    image_name TEXT NOT NULL,
+    FOREIGN KEY (category_id)
+       REFERENCES categories (id) 
+);
+
+CREATE TABLE IF NOT EXISTS categories (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT NOT NULL,
```

Comment:
```suggestion
    name TEXT NOT NULL
```
`,`これがSynatx Errorになる